scripts/new-version-pr-generator.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+const {execSync} = require('child_process');
+const {readFileSync} = require('fs');
+const pathResolve = require('path').resolve;
+const {mkdir} = require('shelljs');
+const yaml = require('js-yaml');
+const semver = require('semver');
+
+const {version: PACKAGE_VERSION} = require('../package.json');
+const secrets = require('../secrets.json');
+const Retry = require('./utilities/retry');
+
+const repositories = ['polaris-styleguide', 'web'];
+
+const YARN_VERSION = yaml
+  .safeLoad(readFileSync(pathResolve(__dirname, '..', 'dev.yml'), 'utf8'))
+  .up.find((obj) => Object.keys(obj).includes('node')).node.yarn;
+
+const root = pathResolve(__dirname, '../');
+const sandbox = pathResolve(root, 'sandbox');
+
+const releaseVersion = `v${PACKAGE_VERSION}`;
+
+function isMajorPrerelease(version) {
+  return (
+    semver.prerelease(version) &&
+    semver.minor(version) === 0 &&
+    semver.patch(version) === 0
+  );
+}
+
+const baseBranch = isMajorPrerelease(PACKAGE_VERSION)
+  ? `v${semver.major(PACKAGE_VERSION)}`
+  : 'master';
+
+const polarisBotName = 'Shopify Polaris Bot';
+const polarisBotEmail = 'shopify-polaris-bot@users.noreply.github.com';
+const polarisBotToken = secrets.github['shopify-polaris'];
+const shopifyPolarisBotGitOverride = `GIT_COMMITTER_NAME='${polarisBotName}' GIT_COMMITTER_EMAIL='${polarisBotEmail}'`;
+
+function pullRequestTemplate(repository, version) {
+  return `
+## Version ${version} of @shopify/polaris just got published!
+
+See what’s new: https://github.com/Shopify/polaris-react/releases/tag/${version}
+
+cc @kaelig @dfmcphee @amrocha @BPScott
```

Comment:
This list of users is relevant for the style guide (although we'll want to remove Dom on Friday), but may not be as relevant for Shopify/web – maybe a few project owners should be pinged instead?

Reply:
I was thinking of maybe creating a channel, then people can leave and join as necessary, do you have any thoughts?

Reply:
I'd look at how "Spy" does this sort of things automatically on some stacks at deploy time. Not sure how to invoke it from a script like this one, but it's where I'd start my research.

Reply:
@amrocha agreed we are going to use the Shopify/polaris-reviewers team. Feel free to join.